DiscordChatExporter.Cli.dockerfile

```
@@ -12,7 +12,9 @@ RUN dotnet publish DiscordChatExporter.Cli -c Release -o ./publish
 # Run
 FROM mcr.microsoft.com/dotnet/runtime:6.0 AS run
 
-COPY --from=build ./publish ./
+COPY --from=build ./publish /opt/dce
 
-WORKDIR ./out
-ENTRYPOINT ["dotnet", "/DiscordChatExporter.Cli.dll"]
\ No newline at end of file
+RUN useradd dce
+USER dce
+WORKDIR /out
```

For consistency. Is this right?

```suggestion
WORKDIR ./out
```

---

In my opinion, is the usage of absolute paths a better practice and is unlikely to change unexpectedly.